src/posthog-core.ts

```diff
@@ -837,7 +834,7 @@ export class PostHog {
     ): CaptureResult | void {
         // While developing, a developer might purposefully _not_ call init(),
         // in this case, we would like capture to be a noop.
-        if (!this.__loaded) {
+        if (!this.__loaded || !this.sessionPersistence || !this._requestQueue) {
```

Comment:
How is it possible for `sessionPersistence` or `_requestQueue` to be false while `__loaded` is true, when both are set in the same method as  `__loaded`?

Reply:
It's not possible but it's pretty hard to get typescript to think that way. Unless you have a handy truck up your sleeve?